paper.tex

```diff
@@ -526,15 +528,30 @@ \section{Population genetics model details}\label{sec:appendix-pop-gen-model}
 \krtcomment{We need to revisit the paragraph(s) below.  We do not assume individuals are exchangeable, because we do not
 	require that all individuals have the same fitness. See, e.g., p 60 of Wakeley for why fitness == exchangeability
 	and why that is required for the standard coalescent.}
-
-Within a population, individuals are assumed to be exchangeable, but there are
-also parameters for nonrandom mating, such as selfing or cloning rates, which
-give the probability that offspring are generated from one generation to the
-next by self-fertilisation or cloning of an individual. Cloning and selfing
-rates can take any value between zero and one, and if both are specified then
-the selfing rate is the probability an offspring is generated through
-self-fertilisation given it was produced sexually (i.e. with probability one
-minus the cloning rate). We omit any details regarding selection.
+\mhcomment{The `exchangeability' assumption also breaks down under 
```

Comment:
We might want to drop the whole "exchangeability" language throughout, since obviously selection is a problem here as well. Although, there is a difference here between the assumption of individuals being exchangeable with outcrossing/selfing/cloning probabilities than sampled genome copies. Within vs between-individual expected TMRCAs are unequal, but the individuals themselves are "exchangeable" in the sense that each has the same probabilities of being selfed vs outcrossed, for example.